app/models/miq_queue.rb

```diff
@@ -121,7 +121,7 @@ def self.put(options)
     options[:args] = [options[:args]] if options[:args] && !options[:args].kind_of?(Array)
 
     msg = MiqQueue.create!(options)
-    msg.warn_if_large_payload
+    msg.warn_if_large_payload unless Rails.env.production?
```

Comment:
to be honest, I think this is only significant in production.
@Fryguy do you remember the back story on this warning?